generation/bootstrap.sh

```diff
@@ -0,0 +1,158 @@
+
+# https://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories
+
+set -xe
+
+[ -z "`git config user.email`" ] && git config --global user.email "${USERNAME:-script}@google.com"
+[ -z "`git config user.name`" ] && git config --global user.name "${USERNAME:-script}"
+
+cd "$(dirname "$0")"
+
+rm -rf monorepo
+mkdir monorepo
+
+cp bootstrap.sh monorepo
+cp repos.txt monorepo
+
+cd monorepo
+
+git clone https://github.com/newren/git-filter-repo.git
+export PATH=$PATH:`pwd`/git-filter-repo
+
+mkdir google-cloud-java
+
+cd google-cloud-java
+git init -b main
+
+cat ../repos.txt | while read service
+do
+  cd ..
+  git clone https://github.com/googleapis/${service}.git
+  cd  ${service}
+  git filter-repo --to-subdirectory-filter ${service}
+
+  # setup owlbot files correctly to match monorepo configuration
+  cp ${service}/.github/.OwlBot.yaml ${service}/.OwlBot.yaml
+  rm ${service}/.github/.OwlBot.lock.yaml
+  rm ${service}/.github/.OwlBot.yaml
+  sed -i.bak '/docker/d' ${service}/.OwlBot.yaml && rm ${service}/.OwlBot.yaml.bak
+  sed -i.bak '/image/d' ${service}/.OwlBot.yaml && rm ${service}/.OwlBot.yaml.bak
+
+  # In monorepo, the staging directory structure tells the destination module to
+  # which the OwlBot Java postprocessor copies the files.
+  sed -i.bak "s|owl-bot-staging|owl-bot-staging/${service}|" ${service}/.OwlBot.yaml && rm ${service}/.OwlBot.yaml.bak
+
+  text=$(grep '^.*api_shortname.*' ${service}/.repo-metadata.json)
+  text=$(echo "$text" | sed 's/\"//g; s/\,//g; s/^[[:space:]]*//' )
+  text=${text/api_shortname/api-name}
+  echo -e "\n"$text>> ${service}/.OwlBot.yaml
+  git add .
+  git config --add secrets.allowed "dest.*src"
+  git commit -am "chore: setup owlbot configuration"
+
+  cd ../google-cloud-java
+  git remote add ${service} ../${service}
+  git config --add secrets.allowed "dest.*src"
+  git fetch ${service} #--tags
+  EDITOR=true git merge --quiet --allow-unrelated-histories ${service}/main
+  git remote remove ${service}
+  rm -rf ../${service}
+done
+
+# cwd: monorepo/google-cloud-java
+echo "Working directory: $(pwd)"
+
+cp ../../../license-checks.xml .
+cp ../../../java.header .
+
+git add --all
+git commit -m 'chore: add template files'
+
+../../print_root_pom.sh > pom.xml
+
+git add pom.xml
+git commit -am 'feat: create aggregator pom'
+
+# Point modules poms and BOMs to the aggregator pom as parent
+bash ../../set_parent_pom.sh
+
+git add --all
+git commit -am 'feat: point modules to the aggregator pom as parent'
+
+../../generate_gapic_bom.sh
+
+# add the gapic bom module to root pom.xml by regenerating aggregator pom
+../../print_root_pom.sh > pom.xml
+
+git add google-cloud-gapic-bom/pom.xml
+git commit -am 'feat: create google-cloud-gapic-bom'
+
+cp ../../gapic_bom_versions.txt google-cloud-gapic-bom/versions.txt
+
+# Generate Release Please configuration files
+bash ../../generate_release_please_config.sh
+
+git add --all
+git commit -am 'feat: create release please configuration'
```

Comment:
Can we change the commit prefix to `chore` ? Otherwise it gets picked up in release-notes.

Reply:
Done.